includes/footer.html

```diff
@@ -60,7 +60,14 @@ <h5 class="white-text fine">coala</h5>
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 2017 coala</span>
+            <span class="left">
+	        ©
+		<script>
+		    document.getElementById("current-year").innerHTML=new Date().getFullYear();
+		</script>
+		<span id="current-year"></span>
+                coala
+            </span>
             <a class="grey-text text-lighten-4 right" href="#!">GNU AGPL v3.0</a>
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpl9zy7y_2/includes/footer.html
+++ b/tmp/tmpl9zy7y_2/includes/footer.html
@@ -65,7 +65,7 @@
 		<script>
 		    document.getElementById("current-year").innerHTML=new Date().getFullYear();
 		</script>
-		<span id="current-year"></span>
+        <span id="current-year"></span>
                 coala
             </span>
             <a class="grey-text text-lighten-4 right" href="#!">GNU AGPL v3.0</a>
```